File: python/assets/resolve.py
import json

from squid_py.ocean.ocean import Ocean

from ..config import CONFIG_FILE
from ..input import INPUT

import logging
logging.getLogger().setLevel(logging.WARNING)

ocean = Ocean(CONFIG_FILE)
ddo = ocean.resolve_did(INPUT)

print("__result__{}".format(json.dumps(ddo.as_dictionary(), indent=2, sort_keys=True)))

File: node/assets/resolve.js
const { Ocean, Logger } = require('@oceanprotocol/squid')
const config = require('../config')
const input = require('../input');

(async () => {
    const ocean = await Ocean.getInstance(config)

    const ddo = await ocean.resolveDID(input)
    Logger.log(`__result__${JSON.stringify(ddo, null, 2)}`)
})()
